File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task126_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.Scanner;

public class RedirectUser {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter 'google' to go to Google, 'facebook' to go to Facebook, or 'exit' to exit: ");
        String userInput = scanner.nextLine();
        if (userInput.equals("google")) {
            System.out.println("Redirecting to Google...");
        } else if (userInput.equals("facebook")) {
            System.out.println("Redirecting to Facebook...");
        } else if (userInput.equals("exit")) {
            System.out.println("Exiting...");
        } else {
            System.out.println("Invalid input. Please try again.");
            main(args);
        }
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task126_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <stdio.h>
#include <string.h>

void redirectUser() {
    char userInput[20];
    printf("Enter 'google' to go to Google, 'facebook' to go to Facebook, or 'exit' to exit: ");
    scanf("%s", userInput);
    if (strcmp(userInput, "google") == 0) {
        printf("Redirecting to Google...
");
    } else if (strcmp(userInput, "facebook") == 0) {
        printf("Redirecting to Facebook...
");
    } else if (strcmp(userInput, "exit") == 0) {
        printf("Exiting...
");
    } else {
        printf("Invalid input. Please try again.
");
        redirectUser();
    }
}

int main() {
    redirectUser();
    return 0;
}